src/host/getset.cpp

```diff
@@ -970,6 +970,37 @@ void DoSrvPrivateBoldText(SCREEN_INFORMATION& screenInfo, const bool bolded)
     buffer.SetAttributes(attrs);
 }
 
+// Method Description:
+// - Retrieves the active ExtendedAttributes of the given screen buffer. Text

```

Comment:
Maybe leave an example of what a few "extended attributes" are here and below to save someone the effort of looking it up. I often say "Attributes, like colors" when writing such comments.